### ISSUE TITLE
No Such Method Error on using a Flux Enhanced Pickaxe

### ISSUE THREAD

**Issue by richardcox13** [CoFH/Feedback#49]
While in enhanced mode, that was enabled before updating to the latest version.

All this on first use of the pick (or any Redstone Arsenal tool after updating to RA 1.1.0RC1-26 and CoFH core 3.0.0B9-99.

Minecraft report:

---- Minecraft Crash Report ----
// Sorry :(

Time: 24/01/15 18:06
Description: Exception in server tick loop

java.lang.NoSuchMethodError: redstonearsenal.item.tool.ItemPickaxeRF.harvestBlock(Lnet/minecraft/world/World;IIILnet/minecraft/entity/player/EntityPlayer;)V
	at redstonearsenal.item.tool.ItemPickaxeRF.func_150894_a(ItemPickaxeRF.java:60)
	at net.minecraft.item.ItemStack.func_150999_a(ItemStack.java:309)
	at net.minecraft.server.management.ItemInWorldManager.func_73084_b(ItemInWorldManager.java:288)
	at net.minecraft.server.management.ItemInWorldManager.func_73082_a(ItemInWorldManager.java:217)
	at net.minecraft.network.NetHandlerPlayServer.func_147345_a(NetHandlerPlayServer.java:498)
	at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:53)
	at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:8)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.7.0_71, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 179798072 bytes (171 MB) / 3165126656 bytes (3018 MB) up to 3165126656 bytes (3018 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx3g -Xms1g -XX:PermSize=128M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 96
	FML: MCP v9.05 FML v7.10.85.1277 Minecraft Forge 10.13.2.1277 103 mods loaded, 103 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	FML{7.10.85.1277} [Forge Mod Loader] (forge-1.7.10-10.13.2.1277.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Forge{10.13.2.1277} [Minecraft Forge] (forge-1.7.10-10.13.2.1277.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	appliedenergistics2-core{rv2-beta-7} [AppliedEnergistics2 Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	CodeChickenCore{1.0.4.29} [CodeChicken Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	<CoFH ASM>{000} [CoFH ASM Data Initialization] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NotEnoughItems{1.0.4.83} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.4.83-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenModsCore{0.6} [OpenModsCore] (minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	debug{1.0} [debug] (denseores-1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	DamageIndicatorsMod{3.2.0} [Damage Indicators] ([1.7.10]DamageIndicatorsMod-3.2.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	appliedenergistics2{rv2-beta-7} [Applied Energistics 2] (appliedenergistics2-rv2-beta-7.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BiblioCraft{1.9.2} [BiblioCraft] (BiblioCraft[v1.9.2][MC1.7.10].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	CoFHCore{1.7.10R3.0.0B9} [CoFH Core] (CoFHCore-[1.7.10]3.0.0B9-99.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Core{6.3.3} [BuildCraft] (buildcraft-6.3.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Forestry{3.3.0.6} [Forestry for Minecraft] (forestry_1.7.10-3.3.0.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BiblioWoodsForestry{1.7} [BiblioWoods Forestry Edition] (BiblioWoods[Forestry][v1.7].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Mantle{1.7.10-0.3.2.jenkins184} [Mantle] (Mantle-1.7.10-0.3.2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Natura{2.2.0} [Natura] (natura-1.7.10-2.2.0.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BiblioWoodsNatura{1.5} [BiblioWoods Natura Edition] (BiblioWoods[Natura][v1.5].jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ThermalFoundation{1.7.10R1.0.0RC1} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.0.0RC1-14.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ThermalExpansion{1.7.10R4.0.0B6} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.0.0B6-16.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BigReactors{0.4.1A2} [Big Reactors] (BigReactors-0.4.1A2.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BinnieCore{2.0-pre5} [Binnie Core] (binnie-mods-2.0-pre5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Botany{2.0-pre5} [Botany] (binnie-mods-2.0-pre5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ExtraBees{2.0-pre5} [Extra Bees] (binnie-mods-2.0-pre5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ExtraTrees{2.0-pre5} [Extra Trees] (binnie-mods-2.0-pre5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Genetics{2.0-pre5} [Genetics] (binnie-mods-2.0-pre5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Transport{6.3.3} [BC Transport] (buildcraft-6.3.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Silicon{6.3.3} [BC Silicon] (buildcraft-6.3.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Builders{6.3.3} [BC Builders] (buildcraft-6.3.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Energy{6.3.3} [BC Energy] (buildcraft-6.3.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Factory{6.3.3} [BC Factory] (buildcraft-6.3.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	BuildCraft|Compat{6.3.0} [BuildCraft Compat] (buildcraft-compat-6.3.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ChickenChunks{1.3.4.16} [ChickenChunks] (ChickenChunks-1.7.10-1.3.4.16-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Railcraft{9.4.0.0} [Railcraft] (Railcraft_1.7.10-9.4.0.0.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ForgeMultipart{1.1.1.320} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.1.320-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	chisel{2.3.3.28} [Chisel 2] (Chisel2-2.3.3.28.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ComputerCraft{1.65} [ComputerCraft] (ComputerCraft1.65.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded{1.7.10R2.8.0RC6} [MineFactory Reloaded] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	EnderIO{1.7.10-2.2.7.325} [Ender IO] (EnderIO-1.7.10-2.2.7.325.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	EnderStorage{1.4.5.27} [EnderStorage] (EnderStorage-1.7.10-1.4.5.27-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ExtraUtilities{1.2.1} [Extra Utilities] (extrautilities-1.2.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	FastCraft{1.12} [FastCraft] (fastcraft-1.12.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	FlatSigns{2.1.0.19} [Flat Signs] (FlatSigns-1.7.10-universal-2.1.0.19.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	funkylocomotion{1.0} [Funky Locomotion] (funky-locomotion-1.7.10-beta-4.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	inpure|core{1.7.10R1.0.0B7} [INpureCore] (INpureCore-[1.7.10]1.0.0B7-19.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	inventorytweaks{1.59-dev-156-af3bc68} [Inventory Tweaks] (InventoryTweaks-1.59-dev-156.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	IronChest{6.0.60.741} [Iron Chest] (ironchest-1.7.10-6.0.60.741-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Waila{1.5.8} [Waila] (Waila-1.5.8a_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	JABBA{1.2.0a} [JABBA] (Jabba-1.2.0a_1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	journeymap{5.0.1} [JourneyMap] (JourneyMap5.0.1_Unlimited_MC1.7.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatAppliedEnergistics{1.7.10R2.8.0RC6} [MFR Compat: Applied Energistics] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatAtum{1.7.10R2.8.0RC6} [MFR Compat: Atum] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatBackTools{1.7.10R2.8.0RC6} [MFR Compat: BackTools] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatBuildCraft{1.7.10R2.8.0RC6} [MFR Compat: BuildCraft] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatChococraft{1.7.10R2.8.0RC6} [MFR Compat: Chococraft] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatExtraBiomes{1.7.10R2.8.0RC6} [MFR Compat: ExtraBiomes] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatForestry{1.7.10R2.8.0RC6} [MFR Compat: Forestry] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatForgeMicroblock{1.7.10R2.8.0RC6} [MFR Compat: ForgeMicroblock] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatIC2{1.7.10R2.8.0RC6} [MFR Compat: IC2] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Mystcraft{0.11.0.00} [Mystcraft] (mystcraft-1.7.10-0.11.0.00.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatMystcraft{1.7.10R2.8.0RC6} [MFR Compat: Mystcraft] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MrTJPCoreMod{1.0} [MrTJPCoreMod] (MrTJPCore-1.7.10-1.0.4.9-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Core{4.5.9.60} [ProjectRed] (ProjectRed-1.7.10-4.5.9.60-Base.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Exploration{4.5.9.60} [ProjectRed-Exploration] (ProjectRed-1.7.10-4.5.9.60-World.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatProjRed{1.7.10R2.8.0RC6} [MFR Compat ProjectRed] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatRailcraft{1.7.10R2.8.0RC6} [MFR Compat: Railcraft] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatRP2{1.7.10R2.8.0RC6} [MFR Compat: RP2] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatSufficientBiomes{1.7.10R2.8.0RC6} [MFR Compat: Sufficient Biomes] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatThaumcraft{1.7.10R2.8.0RC6} [MFR Compat: Thaumcraft] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatThermalExpansion{1.7.10R2.8.0RC6} [MFR Compat: Thermal Expansion] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	TConstruct{1.7.10-1.8.2.build858} [Tinkers' Construct] (TConstruct-1.7.10-1.8.2a.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatTConstruct{1.7.10R2.8.0RC6} [MFR Compat: Tinkers' Construct] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatTwilightForest{1.7.10R2.8.0RC6} [MFR Compat: TwilightForest] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	MineFactoryReloaded|CompatVanilla{1.7.10R2.8.0RC6} [MFR Compat: Vanilla] (MineFactoryReloaded-[1.7.10]2.8.0RC6-40.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	numina{0.4.0.99} [Numina] (Numina-0.4.0.99.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	powersuits{0.11.0.175} [MachineMuse's Modular Powersuits] (ModularPowersuits-0.11.0.175.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons{1.12.3.11} [NEI Addons] (neiaddons-mc1710-1.12.3.11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|Botany{1.12.3.11} [NEI Addons: Botany] (neiaddons-mc1710-1.12.3.11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|Forestry{1.12.3.11} [NEI Addons: Forestry] (neiaddons-mc1710-1.12.3.11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|CraftingTables{1.12.3.11} [NEI Addons: Crafting Tables] (neiaddons-mc1710-1.12.3.11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIAddons|ExNihilo{1.12.3.11} [NEI Addons: Ex Nihilo] (neiaddons-mc1710-1.12.3.11.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	neiintegration{1.0.5} [NEI Integration] (NEIIntegration-MC1.7.10-1.0.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	NEIPlugins{1.1.0.11-unofficial} [NEI Plugins Unofficial] (NEIPlugins-MC1.7.10-1.1.0.11-unofficial.jar.zip) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ObsidiPlates{3.0.0.18} [ObsidiPlates] (ObsidiPlates-1.7.10-universal-3.0.0.18.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenMods{0.6} [OpenMods] (OpenModsLib-1.7.10-0.6.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenPeripheralCore{0.5.0} [OpenPeripheralCore] (OpenPeripheral-1.7.10-AIO-1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenPeripheral{0.2.0} [OpenPeripheralAddons] (OpenPeripheral-1.7.10-AIO-1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenBlocks{1.3} [OpenBlocks] (OpenBlocks-1.7.10-1.3.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenPeripheralIntegration{0.1.0} [OpenPeripheralIntegration] (OpenPeripheral-1.7.10-AIO-1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Transmission{4.5.9.60} [ProjectRed-Transmission] (ProjectRed-1.7.10-4.5.9.60-Integration.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Transportation{4.5.9.60} [ProjectRed-Transportation] (ProjectRed-1.7.10-4.5.9.60-Mechanical_beta.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Compatibility{4.5.9.60} [ProjectRed-Compatibility] (ProjectRed-1.7.10-4.5.9.60-Compat.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Integration{4.5.9.60} [ProjectRed-Integration] (ProjectRed-1.7.10-4.5.9.60-Integration.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Illumination{4.5.9.60} [ProjectRed-Illumination] (ProjectRed-1.7.10-4.5.9.60-Lighting.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ProjRed|Expansion{4.5.9.60} [ProjectRed-Expansion] (ProjectRed-1.7.10-4.5.9.60-Mechanical_beta.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	RedstoneArsenal{1.7.10R1.1.0RC1} [Redstone Arsenal] (RedstoneArsenal-[1.7.10]1.1.0RC1-26.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	TiCTooltips{1.2.1} [TiC Tooltips] (TiCTooltips-mc1.7.10-1.2.1.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	TMechworks{1.7.10-86.e500b88} [Tinkers' Mechworks] (TMechworks-1.7.10-0.2.10.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	Translocator{1.1.1.14} [Translocator] (Translocator-1.7.10-1.1.1.14-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	McMultipart{1.1.1.320} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.1.320-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	denseores{1.0} [Dense Ores] (denseores-1.5.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	ForgeMicroblock{1.1.1.320} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.1.320-universal.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized->Available->Available->Available->Available
	OpenModsLib crash transformers: [gl_capabilities_hook:FINISHED],[player_render_hook:FINISHED],[map_gen_fix:FINISHED],[movement_callback:FINISHED],[stencil_patches:FINISHED]
	AE2 Version: beta rv2-beta-7 for Forge 10.13.2.1230
	Mantle Environment: Environment healthy.
	TConstruct Environment: Environment healthy.
	Stencil buffer state: Function set: GL30, pool: internal, bits: 8
	AE2 Integration: IC2:OFF, RotaryCraft:OFF, RC:ON, BC:ON, MJ6:OFF, MJ5:OFF, RF:ON, RFItem:ON, MFR:ON, DSU:ON, FZ:OFF, FMP:ON, RB:OFF, CLApi:OFF, Waila:ON, InvTweaks:ON, NEI:ON, CraftGuide:OFF, Mekanism:OFF, ImmibisMicroblocks:OFF, BetterStorage:OFF
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 8; [EntityPlayerMP['RichardCox13'/489, l='M1720-One', x=173.70, y=42.00, z=-149.30]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

**Comment by richardcox13** [comment]
Same crash on second usage.

**Comment by marcin212** [comment]
I have the same problem.

**Comment by skyboy** [comment]
Alpha releases available through CurseForge are not to be used in mod packs, and are available to people who do not mind potential world corruption from them.

Stick to the versions listed on Curse (NOT CurseForge) for stable versions.

**Comment by richardcox13** [comment]
>  mod packs

I mentioned no mod pack.

**Comment by skyboy** [comment]
Then don't download alpha versions.

**Comment by eXiGe** [comment]
Who are you releasing these versions for then? How does the "wrong person" submitting the bug report invalidate said report? It defeats the purpose to close the report if the bug still exists. I would have suggested you stipulate for people to download from curse, on curseforge, but you have done the opposite on your website.

"Downloads have been moved to CurseForge." 
http://teamcofh.com/downloads/ near the bottom

I'd like to say more but I am afraid I'd come off as being passive-aggressive and annoyed. Kind of like the manner which nearly everyone is greeted when they try to be helpful by submitting reports here.

**Comment by bochen415** [comment]
I'm with this ^ guy.
It's the golden rule to always post bugs with the latest version.

**Comment by skyboy** [comment]
@eXiGe The bug is resolved by not mixing alphas; the latest core and latest RSA (which I don't recall if it's pushing alphas) are required for this not to crash. It's invalid. 

Further support for alphas doesn't exist: they may corrupt your world, crash constantly, or not even launch. It's the nature of what they are, and people who do not explicitly want to be subject to that are dissuaded from using them because they shouldn't be using them.

**Comment by eXiGe** [comment]
@skyboy The question still remains then, who are you releasing for? If they are possibly broken builds that you want no feedback on, what is the point?

Before this post, I had not seen anything to suggest that I should not use these versions. Your versioning indicates betas and even a release candidate for Thermal Foundation. By your statements, I assume we are to only pay heed to the contradictory icons on CurseForge?

Same with the Curse vs CurseForge statement.

I think some confusion could be avoided here with some communication of your expectations in locations more than closed issues that most won't see.

**Comment by adudney** [comment]
*sigh* He's saying that you should either use the public beta, or use the latest alpha on every CoFH mod at once. Don't mix and match and expect them to still work. The alphas come with an additional qualifier: It may corrupt worlds or do untold other things. Report bugs, they'll see if they're legitimate, and then work on them if so.

TL;DR: Use a single version for ALL the mods. Whether that's the latest alpha or the latest beta is up to you. (But don't expect as much support on alphas, they won't rush if it's not urgent.)

**Comment by richardcox13** [comment]
@skyboy: At the time of this report 3.0.0B9-99.jar was the latest CoFHCore: I updated both together. If the closure is "fixed in CoFHCore >= -100" I have no problem because it clearly documents for everyone how to get the tools back into a useful state.

But please, be accepting of bug reports, even if they are "already fixed in development", I know (I'm stuck with this at work currently) not getting issues reported is worse; more effort required when you find issues in just before going live that are in the test environment.

**Comment by adudney** [comment]
@richardcox13 I apologize, after looking at the specific versions you have listed, you were not guilty of any alpha/beta mixing. I'm thinking the reason he called the modpack out was because the first comment was someone stating they also had this problem, and then referring to this issue from a modpack issue.

So yes, try again with the latest version I suppose. *shrug*
2 things though to improve later bug reports:
1. Put the crash log on pastebin
2. Reduce the modlist to the minimal necessary to reproduce (Thus ensuring that this is or isn't a compatibility issue with another mod)

**Comment by eXiGe** [comment]
@adudney Do you feel like your parroting of information is helpful when you barely grasp that he wasn't "mixing versions" ? I feel like offering you a cracker Polly.

The problem was caused with CoFHCore-[1.7.10]3.0.0B9-98 which was released 6 days ago. It is now on CoFHCore-[1.7.10]3.0.0B9-103. The Redstone Arsenal version to compliment any of those has yet to be released. I would agree with skyboy that it is invalid had the Redstone Arsenal version been released as well. Why not a "fixed in dev" tag until then?

The problem I have is that there is no statement anywhere that I have seen to not use the CurseForge versions. I prefer the CurseForge layout, and judging by the downloads, at least a few hundred others do as well. I know I am not completely out of line in my thinking as they at least changed the statement on their website.

"Downloads have been moved to CurseForge." 
to 
"Downloads have been moved to Curse." 

Why are the downloads sporting beta and release candidates designations in their version name when you intend for them to be alpha's? What you, adudney, fail to realize is that I and the few others that have read this thread understand about the "alpha versions" now, but what about everyone else? Before this, I would have been expecting to download a beta or RC version.

It's up to the COFHTeam if they want to publicly inform a few more of what their expectations are. It makes no difference to me, it can only help them even if it only prevents a few less people from using versions they don't want to be used.
